config/locales/en.yml

```diff
@@ -1211,3 +1211,10 @@ en:
       long: "%B %d, %Y %H:%M"
       short: "%d %b %H:%M"
     pm: pm
+  index_shadow:
+    what_is_your_question: "What's your question?"
+    to_ask_a_question_please: "To ask a question, please"
+    log_in: "log in"
+    or: "or"
+    sign_up: "sign_up"
+    first: "first."
```

Comment:
@aohslund thanks for working on this. There's a question block in line 664 
`index_shadow:` should be indented inside the question block
https://github.com/publiclab/plots2/blob/452a75bbbd1bf9cf8f0ab545a8f6a4eee3ec01a8/config/locales/en.yml#L664

Reply:
Pushed new commit with index_shadow: moved!